lib/src/test/kotlin/at/bitfire/ical4android/validation/FixInvalidDayOffsetPreprocessorTest.kt

```diff
@@ -27,6 +26,14 @@ class FixInvalidDayOffsetPreprocessorTest {
         assertEquals(expected, fixed)
     }
 
+    private fun fixAssertNotEqual(expected: String, testValue: String) {
+        // Fix the duration string
+        val fixed = FixInvalidDayOffsetPreprocessor.fixString(testValue)
+
+        // Assert
+        assertEquals(expected, fixed)
```

Comment:
The method name contains `not equal`, but the assert is `assertEquals`, which is confusing. Shouldn't we call it `fixStringAndAssertEquals` (meaning that it first calls `fixString` and then `assertEquals`)?

Reply:
No, I simply forgot to call `assertNotEquals()` for the assertion. Then the tests fail so we have to change the regex again. See https://github.com/bitfireAT/ical4android/pull/182#discussion_r1829015657